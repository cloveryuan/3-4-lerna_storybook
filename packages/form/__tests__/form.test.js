import MyyForm from "../src/form.vue"
import {mount}  from "@vue/test-utils"

describe('myy-form',()=>{
  test('form-snapshot',()=>{
    const wrapper = mount(MyyForm)
    expect(wrapper.vm.$el).toMatchSnapshot() // 生成快照
  })
})