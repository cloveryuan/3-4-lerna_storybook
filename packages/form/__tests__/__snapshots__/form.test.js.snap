// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`myy-form form-snapshot 1`] = `<form />`;
